spec/integ/irc-to-matrix.spec.js

```diff
@@ -616,7 +617,7 @@ describe("IRC-to-Matrix name bridging", function() {
         cli.joinRoom.and.callFake(function(r, opts) {
             if (count > 0) {
                 count--;
-                return Promise.reject({httpStatus: 504});
+                return Promise.reject({});
```

Comment:
What is this change?

Reply:
It's removing ``httpStatus: 504``, as it is redundant and probably misleading to keep it in.

Reply:
ftr this is the kind of change that should be in it's own PR: It's unrelated and it's impact is unclear on it's own.

Reply:
I think it's unfair to say unrelated, it's the same feature and all we are doing here is unbreaking it. I don't think it's going to be productive to start making sub feature branches unless it gets difficult to read the diffs.